telegram/src/main/kotlin/me/ivmg/telegram/Bot.kt

```diff
@@ -72,7 +72,7 @@ class Bot private constructor(
             false, null -> {
                 val errorMessage: String = when {
                     error?.message != null -> error.message!!
-                    response.errorBody() != null -> response.errorBody().toString()
+                    response?.errorBody() != null -> response?.errorBody().toString()!!
```

Comment:
Can this line be:

```
response?.errorBody() != null -> response.errorBody().toString()
```

Since we have already checked out `response?.errorBody()` is not null, (I think) there is no way `response` is null, right?